packages/default/scss/colorpicker/_layout.scss

```diff
@@ -282,6 +285,19 @@
             text-align: center;
         }
     }
+
+    [dir="rtl"].k-flatcolorpicker {
```

Comment:
What if the rtl direction is set to the whole page and not to the colorpicker widget itself? We also have a class .k-rtl which we use for targeting as well

Reply:
The `k-rtl` class is added to the styles. Since the jquery colorpicker is with different layout than the angular one, we should add feature for styling the jquery colorpicker in RTL mode. This PR includes angular colorpicker rlt styles + added visual tests for both suites. The logged item is here https://github.com/telerik/kendo-themes/issues/808